sql/moz-fx-data-shared-prod/firefox_ios_derived/new_profile_activation_v2/query.sql

```diff
@@ -0,0 +1,60 @@
+-- This table implements the Mobile activation metric defined in June 2022
+-- Based on data from the first seven days since profile creation, we select
+-- clients with at least three days of use (first open plus two) and who
+-- perform a search on the later half of the week. As such, this table will
+-- always lag new profiles by seven days and the CTEs are filtered for
+-- corresponding periods.
+-- Each entry in this table corresponds to a new_profile
+WITH dou AS (
+  SELECT
+    client_id,
+    ARRAY_LENGTH(
+      mozfun.bits28.to_dates(mozfun.bits28.range(days_seen_bits, -5, 6), submission_date)
+    ) AS days_2_7,
+  FROM
+    firefox_ios.baseline_clients_last_seen
+  WHERE
+    submission_date = @submission_date
+    AND DATE_DIFF(submission_date, first_seen_date, DAY) = 6
+),
+client_first_seen AS (
```

Comment:
We have `first_seen_date` in `baseline_clients_last_seen`, I believe we can drop this and just get it from there